internal/crowdactions/crowdaction.go

```diff
@@ -0,0 +1,81 @@
+package crowdaction
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/CollActionteam/collaction_backend/internal/constants"
+	"github.com/CollActionteam/collaction_backend/internal/models"
+	"github.com/CollActionteam/collaction_backend/utils"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/aws/aws-sdk-go/service/dynamodb/expression"
+)
+
+const Separator = "### app version:"
+
+type DynamoRepository interface {
+	GetDBItem(tableName string, pk string, crowdactionId string) (map[string]*dynamodb.AttributeValue, error)
```

Comment:
``the repository should not export adapted types,
you can alias the type or create a specific type and convert  `dynamodb.AttributeValue` to it.



note: creating an alias for an external type is like this:

```Go
type AttributeValue = dynamodb.AttributeValue
```

but creating an internal type and converting it is better and less error-prone.